cvdupdate/cvdupdate.py

```diff
@@ -432,6 +428,39 @@ def _query_dns_txt_entry(self) -> bool:
 
         return got_it
 
+    def _get_nameserver_configuration(self) -> list[str]:
```

Comment:
I get an error when I use it:

```

❯ CVDUPDATE_NAMESERVER=1.1.1.1,8.8.8.8 cvd update
Traceback (most recent call last):
  File "/home/micasnyd/.local/bin/cvd", line 11, in <module>
    load_entry_point('cvdupdate', 'console_scripts', 'cvd')()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 490, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2854, in load_entry_point
    return ep.load()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2445, in load
    return self.resolve()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2451, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/home/micasnyd/workspace/cvdupdate/cvdupdate/__main__.py", line 42, in <module>
    from cvdupdate import auto_updater
  File "/home/micasnyd/workspace/cvdupdate/cvdupdate/auto_updater.py", line 3, in <module>
    from cvdupdate.cvdupdate import CVDUpdate
  File "/home/micasnyd/workspace/cvdupdate/cvdupdate/cvdupdate.py", line 45, in <module>
    class CVDUpdate:
  File "/home/micasnyd/workspace/cvdupdate/cvdupdate/cvdupdate.py", line 431, in CVDUpdate
    def _get_nameserver_configuration(self) -> list[str]:
TypeError: 'type' object is not subscriptable
```

It looks like the `list[str]` only works with Python 3.9+. For compatibility with Python 3.5-3.8, according to https://stackoverflow.com/questions/31905597/how-to-properly-function-annotate-type-hint-a-list-of-strings, 
the type annotation here should be `-> List[str]:`